src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagService.cs

```
@@ -47,5 +56,90 @@ public async Task<AddCustomTagResponse> AddCustomTagAsync(IEnumerable<CustomTagE
             // Current solution is synchronouse, no job uri is generated, so always return emtpy.
             return new AddCustomTagResponse(job: string.Empty);
         }
+
+        public async Task DeleteCustomTagAsync(string tagPath, CancellationToken cancellationToken = default)
+        {
+            DicomTag[] tags;
+            if (!_dicomTagParser.TryParse(tagPath, out tags, supportMultiple: false))
+            {
+                throw new InvalidCustomTagPathException(
+                    string.Format(CultureInfo.InvariantCulture, DicomCoreResource.InvalidCustomTagPath, tagPath == null ? string.Empty : tagPath));
+            }
+
+            string path = tags[0].GetPath();
+
+            CustomTagStoreEntry customTagStoreEntry = await _customTagStore.GetCustomTagAsync(path, cancellationToken);
```

add comments -> GetCustomTagAsync should throw exception when not found